src/components/Layout.tsx

```diff
@@ -16,21 +16,23 @@ export const Layout: VFC<FlexProps> = ({ children, ...rest }) => {
   const { chain } = useNetwork()
 
   const router = useRouter()
-  const isStrategiesLandingPage =
-    router.pathname.split("/")[1]?.toLowerCase() === "strategies" &&
-    router.pathname.split("/")[3]?.toLowerCase() !== "manage"
+
+  const isHomeOrStrategiesLandingPage =
+    router.pathname === "/" ||
+    (router.pathname.split("/")[1]?.toLowerCase() === "strategies" &&
```

Comment:
For these last two lines, could we not use something like `router.pathname.includes("strategies")`, or do you feel that wouldn't be specific enough? [Example docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
`Array.includes` may also cover future cases where routing patterns change for whatever reason 👀

Reply:
Thanks Xan, I think this implementation is much better 👍